docs/search/simplified-query-writing.md

```diff
@@ -12,53 +12,64 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+This feature is in beta, with select features available for early access. Full general availability is expected by the end of the year. To participate, contact your Sumo Logic account executive.
 
-Our new **Simplified Query Writing** enhancements focus on delivering a frictionless query experience with real-time syntax suggestions, schema suggestions, and partial/full query predictions. These features make it easier for both novice and advanced users to build accurate and efficient queries.
+Our new simplified query writing enhancements provide a frictionless experience with real-time syntax suggestions, schema suggestions, and partial/full query predictions. These features simplify the query-building process, helping both novice and advanced users write accurate, efficient queries.
 
-## Key features
+By incorporating user feedback, we’ve implemented improvements to reduce the complexity of Sumo Logic’s query language, making it easier to discover relevant fields, minimize errors, and benefit from intelligent query-building assistance.
 
-* **Syntax suggestions**. Get real-time syntax suggestions as you type, reducing errors and speeding up query creation.
-* **Schema suggestions**. Automatically receive schema-related suggestions, helping you navigate complex data structures like JSON logs with ease.
-* **Partial/Full query suggestions**. Based on your input, the system predicts and suggests the next operator or even a full query, streamlining the process.
-* **Frictionless query experience**. An intuitive interface highlights errors and provides immediate fixes, allowing for smooth and seamless query building.
-* **Key-Value pair support**. Offers enhanced suggestions for structured logs like JSON, recognizing keys and values for more precise filtering.
-* **Facet identification for JSON Logs**. Identifies key facets (e.g., timestamp, status) to help refine your queries.
+## Key enhancements made to the query-building experience
 
-## How to use
+* **Improved learning curve**. We’ve simplified learning and using Sumo Logic queries. The new syntax suggestions and schema-based prompts allow you to write queries faster with less need for deep familiarity with the syntax.
+* **Enhanced auto-complete**. The auto-complete function has been optimized to provide more relevant, context-based suggestions that align with your queries and organizational patterns, improving accuracy and efficiency.
+* **Streamlined field discovery**. No need to manually search for fields within your logs. The system automatically suggests key fields, especially for structured data like JSON, making it easier to navigate complex data and build accurate queries.
 
-### 1. Start a Log Search
 
-From the [**Classic UI**](/docs/get-started/sumo-logic-ui-classic) or the [**New UI**](/docs/get-started/sumo-logic-ui), go to **Log Search** to start crafting your query.
+## Token-by-token prediction and auto-complete
```

Comment:
I'd place all these sections that describe the benefits of the feature into a section labelled something like "Benefits".